security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/JsonOperatorStrategyList.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.json;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.powsybl.commons.json.JsonUtil;
+import com.powsybl.security.operator.strategy.OperatorStrategyList;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Objects;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+public final class JsonOperatorStrategyList {
```

Comment:
I would remove this class and move all this utility methods to classes: OperatorStrategy.readJson